tests/client/client_common.py

```diff
@@ -140,3 +142,27 @@ def test_build_and_run_instance(ebnt, container_name):
     time.sleep(.1)
 
     assert not instance.is_running()
+
+
+@pytest.mark.docker
+@pytest.mark.skipif(not has_docker(), reason='no docker installed')
+def test_build_and_delete_image(ebnt: Ebonite, model: Model):
+    task = ebnt.get_or_create_task('test_proj', 'test_task')
+    model = ebnt.push_model(model, task)
+    try:
+        with create_docker_client() as client:
+            client.images.remove('test_image')
+    except ImageNotFound:
+        pass
+
+    image = ebnt.build_image('test_image', model)
+
+    assert ebnt.get_image(image.name, model) == image
+
+    with create_docker_client() as client:
+        assert client.images.get(image.name) is not None
```

Comment:
if I remeber correctly, in case of missing image it throws something like `docker.errors.ImageNotFound`